wordpress-mobile/WordPress-iOS
# Add notifications inline action for share an own published post

Fixes #22473

## Description

This PR adds an inline action to share a post.

## Test Instructions

* Log in to the Jetpack app (with an account containing various notifications)
* Tap the notifications tab
* Scroll to a notification for a post like
* Expect to see a share icon
* Tap the share icon
* Expect the share activity to launch
* Expect this event to be tracked `notifications_inline_action_tapped <inline_action: share_post>`

Test for regression:

* Tap the other part of the notification
* Expect the notification detail view to be presented (as usual)

## Regression Notes
1. Potential unintended areas of impact
Notification cell configuration logic was refactored, so double check the notification content is correct.

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Manual testing.

3. What automated tests I added (or what prevented me from doing so)
None.

## PR submission checklist:

- [x] have completed the Regression Notes.

Testing checklist:
